### PR TITLE
Create BLorient13278

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record according to S. Strelcyns description. I was able to identify the msItems, except for ms_i2, for which I propose a new LIT ID, as I have outlined in https://github.com/BetaMasaheft/Documentation/issues/2520.

Unfortunately, I made a mistake and created a record called BL13278 instead of BLorient13278 as it should be. I have created another new record with the correct name. The outdated earlier issue can be ignored in this pull request, but should be deleted in a new pull request immediately after merging this one. If I remember correctly, a new pull request is needed, otherwise the deletion will cause chaos in the application. Is this correct?

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient13278.xml [hunk 1]
```diff
@@ -0,0 +1,227 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient13278" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient13278.xml:8]: ya-

File: LondonBritishLibrary/orient/BLorient13278.xml [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+                            <locus from="3r" to="39r"/>
+                            <title ref="LIT1099Anapho"/>
+                            <incipit xml:lang="gez">አኰቴተ፡ <supplied reason="undefined">ቍርባን፡</supplied> ዘእግዝእትነ፡ ማርያም፡ ዘደረሰ፡ አባ፡ ሕርያቆስ፡ 
+                                <supplied reason="undefined">ኤጲስ፡ ቆጶስ፡ ዘሀገረ፡ ብሕንሳ፡</supplied> በመንፈስ፡ ቅዱስ፡ በእንተ፡ እግዝእትነ፡ ማርያም፡ ወላዲተ፡ አምላክ፡ </incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="39v" to="88v"/>
+                            <title ref="LIT0000000000000000000000000">Yaḥǝgǝnnā yaśǝrʿat maṣḥaf</title>
+                            <note>Amharic version of the <ref type="work" corresp="LIT1968Mashaf">Maṣḥafa śǝrʿat</ref>.</note>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> በዚህቺ፡ በቅድስት፡ ሥላሴ፡ ሠራነ፡ መስሪነታቺው፡ ፩አምላክ፡ በሚባል፡ አንደነታቺው፡ አብ፡
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient13278.xml:44]: am

File: LondonBritishLibrary/orient/BLorient13278.xml [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 13278</idno>
+                        <altIdentifier><idno>Strelcyn 39</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient13278.xml:34]: the element textLang can be added to every msItem

File: LondonBritishLibrary/orient/BLorient13278.xml [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> በዚህቺ፡ በቅድስት፡ ሥላሴ፡ ሠራነ፡ መስሪነታቺው፡ ፩አምላክ፡ በሚባል፡ አንደነታቺው፡ አብ፡ 
+                                ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ በሚባል፡ ፫፡ እነታቸው፡ ፫አካል፡ ፩ህልውና፡ ፩ግብር፡ ፩ባሕርያ፡ ፩አምላክ፡ በሆነ፡ ሰሪናቸው፡ አነ፡ ውእቱ፡ አምላከ፡ አብርሃም፡ ወአምላከ፡ 
+                                ይስሐቅ፡ ወ<pb n="40r"/>አምላከ፡ ያዕቆብ፡ ቢሚባሉ፡ ሥላሴ፡ <gap reason="ellipsis"/> ዛቲ፡ መጽሐፈ፡ ሥርዓት፡ ዘወጽአት፡ እመንበረ፡ ማርቆስ፡ ሐዋርያ፡ 
+                                <pb n="41v"/>እመካነ፡ ማልቃ፡ ዘምስር፡ ማለት፡ ይህቿ፡ የሕግና፡ የሥርዓት፡ መጽሐፍ፡ የምስር፡ ወገን፡ ከምትሆን፡ ከማልቃ፡ ቦታ፡ ተነሥታ፡ በሐዋርያው፡ በማርቆስ፡ 
+                                ቦታ፡ የተገንቲ፡ ናት፡ ሲል፡ ነው፡ ዘአንበሩ፡ መምሕራን፡ ሊቃውንት፡ ክቡራን፡ መጋብያ፡ ምስጢር፡ ርቱዓነ፡ ሃይማኖት፡ እለኒኸውም፡ ያስቀመጥዋት፡ ሰወቺ፡ ምግባር፡ 
+                                ተሃ<pb n="42r"/>ይማኖት፡ ሃይማኖት፡ ተመግባር፡ አስተባብረው፡ የያዙ፡ ናቸው፤ </incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="88v" to="91v"/>
+                            <title>Three miracles of the Virgin Mary</title>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient13278.xml:53]: LIT3585Taamme

**Review by thea-m — CHANGES_REQUESTED** [review]
Nice, thank you!

**Comment by thea-m** [comment]
My understanding here would be that the wrongly created record can be in this case deleted before the branch is merged

**Comment by eu-genia** [comment]
If a record was created in Oxygen only (without the app) it can be always renamed/deleted/moved at any time, without waiting to merge first. This is actually recommended to limit the traffic to the server.

ONLY records created through the app MUST be synced first.

File: LondonBritishLibrary/orient/BLorient13278.xml [hunk 5]
```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient13278" xml:lang="en" type="mss">
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient13278.xml:5]: if you created the first version directly in Oxygen there was no need to create a copy, you could simply rename the file and update the `xml:id`

if you created the first version in the APP then it must be first saved in the `new `folder and then synced first

File: LondonBritishLibrary/orient/BLorient13278.xml [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Not very careful hand.</desc>
+                                <date notBefore="1900" notAfter="1978"/>
+                            </handNote>                           
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <locus target="#1r"/>
+                                <desc><ref type="authFile" corresp="AT1041HolyMen">Miniature of
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient13278.xml:110]: NB every `corresp `= hyperlink
Now you have one hyperlink (`persName`) inside of another (`ref`), the computer cannot process this.
Please do not overlap.
either close the `ref `immediately `<ref type="authFile" corresp="AT1041HolyMen"/> `  or at least before `persName `starts

File: LondonBritishLibrary/orient/BLorient13278.xml [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+                                <q xml:lang="gez">ፍቁረ፡ እግዚእ፡</q>
+                            </decoNote>
+                            <decoNote xml:id="d5" type="miniature">
+                                <locus target="#2r"/>
+                                <desc><ref type="authFile" corresp="AT1031VirginAndChild">Miniature of the Virgin with the Child</ref>.</desc>
+                                <q xml:lang="gez">ምስለ፡ ፍቁር፡ ወልዳ፡</q>
+                            </decoNote>
+                            <decoNote xml:id="d6" type="miniature">
+                                <locus target="#2r"/>
+                                <desc><ref type="authFile" corresp="AT1082Angel">Miniature of the
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient13278.xml:136]: as above

File: LondonBritishLibrary/orient/BLorient13278.xml [hunk 8]
```diff
@@ -0,0 +1,232 @@
...
+                                    <persName ref="PRS7049Michael">Archangel Michael</persName></ref> on the left of the page.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d7" type="miniature">
+                                <locus target="#2r"/>
+                                <desc><ref type="authFile" corresp="AT1082Angel">Miniature of the 
+                                    <persName ref="PRS4377Gabriel">Archangel Gabriel</persName></ref> on the right of the page.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d8" type="miniature">
+                                <locus target="#2v"/>
+                                <desc>Empty space intended for a <ref type="authFile" corresp="AT1082Angel">miniature of the
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient13278.xml:146]: as above

File: LondonBritishLibrary/orient/BLorient13278.xml [hunk 9]
```diff
@@ -0,0 +1,232 @@
...
+                <keywords>
+                    <term key="ChristianContent"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="NewTestament"/>
+                    <term key="AmharicLiterature"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient13278.xml:220]: `am`

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by thea-m — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**